bin/npmbrew-clean

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+require(__dirname + "/../lib/").clean()
```

Comment:
I'd like to keep bin simple. Could you please put only the following as npmbrew.js ?
```js
require(__dirname + "/../lib/cli")
```